MANIFEST.in

```diff
@@ -1,2 +1,3 @@
 include *.txt
 include *.md
+include persim/landscapes/*
```

Comment:
This is surprising. Is this necessary? I thought setuptools will automatically include any `*.py` files.